### PR TITLE
deduping gov docs + creating a lifecycle checklist

### PR DESCRIPTION
**Pull request by parispittman**
This addresses several related issues 
(deduping, removing tactical/procedure from policy, formatting) that I won't completely tag here for purposes of not auto closing the issues. 1997, 2857

This needs many eyes. I tried to correct some content errors that I saw but please leave comments on stale processes, things that are duplicated still, etc. 

TODO:
- [ ] generator/list.tmpl -> need to change the sig creation link to sig-wg-lifecycle (doc added here)
- [ ] fix all dependancies to the root /sig-goverance.md to direct to the /committee-steering/sig-governance file.

### CODE REVIEW

**Comment by bgrant0607** [comment]
cc @pwittrock

File: governance.md [hunk 1]
```diff
@@ -97,10 +105,10 @@ field technical escalations, etc.
 Example subprojects for a few SIGs:
 * SIG Network: pod networking (CNI, etc.), Service (incl. kube-proxy),
 Ingress, DNS, and Network policy
-* SIG Apps: workload APIs, Helm, Kompose, ...
+* SIG Apps: workload APIs, Kompose, ...
```
Comment (bgrant0607) [inline governance.md]: Please just remove SIG Apps as an example. The other examples are more clear

File: sig-governance.md [hunk 2]
```diff
@@ -1,114 +1,3 @@
-# SIG Governance
+This document should be deleted once all dependancies have been dealt with. This document was merged with several but most notably sunsetted in favor of the work in the [committee-steering] folder.
```
Comment (cblecker) [inline sig-governance.md]:
```
W0221 03:25:09.483] sig-governance.md:1:41: "dependancies" is a misspelling of "dependencies"
```

File: committee-steering/governance/sig-governance.md [hunk 3]
```diff
@@ -1,16 +1,24 @@
...
+In order to standardize Special Interest Group efforts, create maximum transparency, and route contributors to the appropriate SIG, SIGs should follow these guidelines:
+
+- Create a charter and have it approved according to the [SIG charter process]
+- Meet regularly, at least for 30 minutes every 3 weeks, except November and December
+- Keep up-to-date meeting notes, linked from the SIG's page in the community repo
+- Record meetings and make them publicly available on a YouTube playlist
+- Report activity in the weekly community meeting at least once every quarter
+- Participate in release planning meetings and retrospectives, and burndown meetings, as needed
+- Ensure related work happens in a project-owned github org and repository, with code and tests explicitly owned and supported by the SIG, including issue triage, PR reviews, test-failure response, bug fixes, etc.
+- Use the [forums provided] as the primary means of working, communicating, and collaborating, as opposed to private emails and meetings
```
Comment (pwittrock) [inline committee-steering/governance/sig-governance.md]: Should we add something about having a clearly defined group of deciders for sub projects?

File: committee-steering/governance/sig-governance.md [hunk 4]
```diff
@@ -91,6 +99,11 @@ Subproject Owner Role.  (this different from a SIG or Organization Member).
 - Contributing instructions defined in the SIG CONTRIBUTING.md
 
 ### Project Management
+In addition, SIGs have the following responsibilities to SIG PM:
```
Comment (pwittrock) [inline committee-steering/governance/sig-governance.md]: SIGs?  Subprojects?

File: committee-steering/governance/wg-governance.md [hunk 5]
```diff
@@ -4,14 +4,14 @@
 Working Groups provide a formal avenue for disparate groups to collaborate around a common problem, craft a balanced
 position, and disband. Because they represent the interests of multiple groups, they are a vehicle for consensus
 building.  If code is developed as part of collaboration within the Working Group, that code will be housed in an
-appropriate repository as described in the [repositories document][repodoc].  The merging of this code into the repository
```
Comment (pwittrock) [inline committee-steering/governance/wg-governance.md]: This breaks this link.

File: committee-steering/governance/wg-governance.md [hunk 6]
```diff
@@ -4,14 +4,14 @@
...
+appropriate repository as described in the [repositories document].  The merging of this code into the repository
 will be governed by the standard policies regarding submitting code to that repository (e.g. developed within one or
 more Subprojects owned by SIGs).
 
 Because a working group is an official part of the Kubernetes project it is subject to steering committee oversight
 over its formation and disbanding.
 
-[repodoc]: https://github.com/kubernetes/community/blob/master/github-management/kubernetes-repositories.md
+The tactical checklist to creation and/or retirement is found in the [SIG / WG lifeycle] document.
```
Comment (pwittrock) [inline committee-steering/governance/wg-governance.md]: This breaks the link

File: committee-steering/governance/wg-governance.md [hunk 7]
```diff
@@ -73,23 +73,7 @@ should eventually be reflected in a pull request on sigs.yaml:
...
-- meeting information
-- contact methods
-- any [sig](sig-governance.md) stakeholders
-
-The pull request should be labeled with any SIG stakeholders and committee/steering. And since GitHub notifications
-are not a reliable means to contact people, an email should be sent to the mailing lists for the stakeholder SIGs,
-and the steering committee with a link to the PR. A member of the community admin team will place a /hold on it
-until it has an LGTM from at least one chair from each of the stakeholder SIGs, and a simple majority of the steering
-committee.
+Once the above questions have been answered, complete the rest of the checklist in the [SIG / WG lifeycle] document
```
Comment (pwittrock) [inline committee-steering/governance/wg-governance.md]: Link for [SIG / WG lifeycle] doesn't work

File: committee-steering/governance/wg-governance.md [hunk 8]
```diff
@@ -117,7 +101,7 @@ within the Working Group, and [sigs.yaml](/sigs.yaml) should be updated.
 References
 
 - [1] https://github.com/kubernetes/community/pull/1994
-- [2] https://groups.google.com/a/kubernetes.io/d/msg/steering/zEY93Swa_Ss/C0ziwjkGCQAJ 
-
+- [2] https://groups.google.com/a/kubernetes.io/d/msg/steering/zEY93Swa_Ss/C0ziwjkGCQAJ
 
+[SIG / WG lifeycle]
```
Comment (pwittrock) [inline committee-steering/governance/wg-governance.md]: Must add the target of the link here.

File: sig-governance.md (hunk 2 above)
Comment (pwittrock) [inline sig-governance.md]: > This document was merged with several but most notably sunsetted in favor of the work

Awkward sentence.

File: sig-wg-lifecycle.md [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+  - any sig stakeholders  
+  - any subproject stakeholders
+- [ ] Add SIG-related docs like charter.md, schedules, roadmaps, etc. to your new kubernetes/community/SIG-foo directory once the above PR is merged.
+- [ ] File a [Kubernetes/Org] Issue for a label; read about our [GitHub management] services   
+
+## Communicate:
+Each one of these has a linked canonical source guideline from set up to moderation and your role and responsibilities for each. We are all responsible for enforcing our [code of conduct].  
+- [ ] Read [moderation.md] and understand your role in keeping our community safe
+- [ ] Create your mailing lists - One for your members and another for your chairs/leads
+- Either [GoogleGroups] OR [discuss.kubernetes.io]
```
Comment (pwittrock) [inline sig-wg-lifecycle.md]: discuss.kubernetes.io link doesn't work

File: sig-wg-lifecycle.md [hunk 10]
```diff
@@ -0,0 +1,116 @@
...
+- [ ] Request a slack channel. [slack-guidelines.md]
+- [ ] Request a YouTube playlist link [k8syoutubecollaboration.md]
+- [ ] Request a zoom account [zoom-guidelines.md]
+
+## Engage:
+...as a chair/tech lead with other chairs/tech leads
+- [ ] Subscribe to the kubernetes-sig-leads@googlegroups.com group
+- [ ] Join the #chairs-and-techleads slack channel  
+
+...with the community as part of [sig-governance]
```
Comment (pwittrock) [inline sig-wg-lifecycle.md]: sig-governance link doesn't work

File: sig-wg-lifecycle.md [hunk 11]
```diff
@@ -0,0 +1,116 @@
...
+## Prerequisites for a SIG
+- [ ] Read [sig-governance.md]
+- [ ] Send an email to the Steering Committee <steering@kubernetes.io> to scope the SIG and get provisional approval.
+- [ ] Look at the checklist below for processes and tips that you will need to do while this is going on. It's best to collect this information upfront so you have a smoother process to launch
+- [ ] Follow the [SIG charter process] to propose and obtain approval for a charter
+- [ ] Announce new SIG on kubernetes-dev@googlegroups.com
+
+## Prerequisites for a WG
+- [ ] Read [wg-governance.md]
+- [ ] Send email to [kubernetes-dev@googlegroups.com] with some of the questions answered from wg-goverance.md and wait for community discourse; ask for SIG sponsorship
```
Comment (pwittrock) [inline sig-wg-lifecycle.md]: I don't think the '[' are required.  It renders them.

Comment (pwittrock) [inline sig-wg-lifecycle.md]: Should we have a template for this so they are easy to spot.  e.g. the Subject is `WG-Creation-Request: WG Foo`

File: sig-wg-lifecycle.md [hunk 12]
```diff
@@ -0,0 +1,116 @@
...
+- [ ] Do the first checklist item in the #GitHub section below and add a row to the WG section:
+  - [ ] Label with committee/steering and wait for a simple majority
+  - [ ] Also add sponsoring SIG Chair/Tech Leads as approvers; you'll get this from the community email above
```
Comment (pwittrock) [inline sig-wg-lifecycle.md]: I don't think WG's have Tech Leads.

File: sig-wg-lifecycle.md [hunk 13]
```diff
@@ -0,0 +1,116 @@
...
+  - [ ] Place a `/hold` on it until the members that need to review have; a contributor experience member will do this for you if they don't see one already  
+- [ ] Send an email to the stakeholder SIG mailing lists and steering committee with the sigs.yaml pull request
+
+## GitHub:
+- [ ] Submit a PR that will add rows to [sigs.yaml] using the [generator doc]; this will create README files and OWNERS_ALIASES files for your new directory in `kubernetes/community`
+- You’ll need:  
+  - SIG Name  
+  - Directory url  
+  - mission statement
```
Comment (pwittrock) [inline sig-wg-lifecycle.md]: nit: inconsistent capatilization

File: committee-steering/governance/sig-governance.md (hunk 4 above)
Comment (parispittman) [reply committee-steering/governance/sig-governance.md]: @justaugustus @jdumars @calebamiles ^^

File: committee-steering/governance/sig-governance.md [hunk 14]
```diff
@@ -155,11 +168,15 @@ Issues impacting multiple subprojects in the SIG should be resolved by either:
   - after 3 or more months it *SHOULD* be retired
   - after 6 or more months it *MUST* be retired
 
+[SIG PM]: https://github.com/kubernetes/community/tree/master/sig-pm
+[k/features]: https://github.com/kubernetes/features
```
[the remark this answers is not in the document]
Comment (parispittman) [reply committee-steering/governance/sig-governance.md]: ty

File: committee-steering/governance/sig-governance.md (hunk 3 above)
Comment (parispittman) [reply committee-steering/governance/sig-governance.md]: wouldn't hurt

**Comment by parispittman** [comment]
@pwittrock PTAL

File: committee-steering/governance/sig-governance.md (hunk 4 above)
Comment (pwittrock) [reply committee-steering/governance/sig-governance.md]: Non-blocking comment

File: sig-wg-lifecycle.md [hunk 15]
```diff
@@ -0,0 +1,117 @@
...
+## Prerequisites for a SIG
+- [ ] Read [sig-governance.md]
+- [ ] Send an email to the Steering Committee <steering@kubernetes.io> to scope the SIG and get provisional approval.
+- [ ] Look at the checklist below for processes and tips that you will need to do while this is going on. It's best to collect this information upfront so you have a smoother process to launch
+- [ ] Follow the [SIG charter process] to propose and obtain approval for a charter
+- [ ] Announce new SIG on kubernetes-dev@googlegroups.com
+
+## Prerequisites for a WG
+- [ ] Read [wg-governance.md]
+- [ ] Send email to [kubernetes-dev@googlegroups.com] titled "WG-Creation-Request: WG Foo" with some of the questions answered from wg-goverance.md and wait for community discourse; ask for SIG sponsorship
```
Comment (pwittrock) [inline sig-wg-lifecycle.md]: should wg-governance.md be a link?

**Comment by pwittrock** [comment]
Overall this is a huge improvement.  Thank you.

/lgtm
/approve

**Comment by bgrant0607** [comment]
Thanks much.
/hold cancel

**Pull request merged**